src/org/labkey/test/tests/query/QueryMetadataTest.java

```diff
@@ -0,0 +1,214 @@
+                "  <table tableName=\"queryMetadataTestList\" tableDbType=\"NOT_IN_DB\">\n" +
+                "    <columns>\n" +
+                "      <column columnName=\"value\">\n" +
+                "        <description>has new description</description>\n" +
+                "      </column>\n" +
+                "    </columns>\n" +
+                "  </table>\n" +
+                "</tables>";
+
+        var queryPage = editPage.clickEditSource();
+        checker().withScreenshot("xml_mismatch")
+                .verifyEquals("expect xml to show only the delta for value description",
+                expectedXml, queryPage.getMetadataXml());
+    }
+
+    /*
+        EditMetadataPage allows you to edit multiple field properties
+        ensure that 'reset to default' from the editMetadataPage
+     */
+    @Test
+    public void testResetToDefault()
+    {
+        var editPage = EditMetadataPage.beginAt(this, getProjectName(), "lists", TEST_LIST);
+        editPage.fieldsPanel()
+                .getField("date")
+                .setDescription("has new description");
+        editPage.fieldsPanel()
+                .getField("value")
+                .setLabel("ValueLabel");
+        editPage.clickSave();
+        // verify/confirm success
+        editPage.waitForSuccess();
+        var queryPage = editPage.clickEditSource();
+        String expectedXml = "<tables xmlns=\"http://labkey.org/data/xml\">\n" +
+                "  <table tableName=\"queryMetadataTestList\" tableDbType=\"NOT_IN_DB\">\n" +
+                "    <columns>\n" +
+                "      <column columnName=\"value\">\n" +
+                "        <columnTitle>ValueLabel</columnTitle>\n" +
+                "      </column>\n" +
+                "      <column columnName=\"date\">\n" +
+                "        <description>has new description</description>\n" +
+                "      </column>\n" +
+                "    </columns>\n" +
+                "  </table>\n" +
+                "</tables>";
+        var actualXml = queryPage.getMetadataXml();
+        checker().withScreenshot("initial_edits")
+            .wrapAssertion(()-> assertThat(actualXml)
+            .as("expect xml to show only the delta for value description")
+            .isEqualTo(expectedXml));
+        queryPage.clickSave();
+        queryPage.goBack();
+
+        editPage = new EditMetadataPage(getDriver());
+
+        // now ensure that 'reset to default' clears the xml delta
+        editPage.resetToDefault();
+        queryPage = editPage.clickEditSource();
+
+        String expectedAfterXml =
+                "<tables xmlns=\"http://labkey.org/data/xml\">\n" +
+                        "  <table tableName=\"queryMetadataTestList\" tableDbType=\"NOT_IN_DB\">\n" +
+                        "    <columns>\n" +
+                        "    </columns>\n" +
+                        "  </table>\n" +
+                        "</tables>\n";
```

Comment:
Use text blocks for stuff like this. IntelliJ should offer to do the refactor automatically.
```suggestion
        String expectedAfterXml = """
                <tables xmlns="http://labkey.org/data/xml">
                  <table tableName="queryMetadataTestList" tableDbType="NOT_IN_DB">
                    <columns>
                    </columns>
                  </table>
                </tables>
                """;
```